index.md

```diff
@@ -72,16 +72,16 @@ can either be static or implement synaptic plasticity rules such as
 * [Synapses in NEST: An overview](synapses_overview)
 * [Developing synapse models](synapse_models)
 
-## Updating models from 2.14 or prior to 2.16
+## Updating Models for the 5g Kernel
```

Comment:
@steffengraber "5g kernel" is a term we are using only internally, so it may confuse users, so we should use 2.14 and 2.16 here, and similarly below for "4g". To avoid having the same line twice, once as heading and once as bullet point, I'd suggest to change the text in the bullet point to "How to update models from 2.14 or prior to 2.16" and corresponding for 4g. Sorry I didn't make an explicit suggestion earlier.

Reply:
Ok, now it is:

    ## Updating models to 2.16
    * [How to update models from NEST 2.14 or prior to 2.16](model_conversion_5g)

    ## Updating models to 2.6 or later
    * [How to update models from NEST 2.4 or prior to 2.6 or later](model_conversion_3g_4g)